core/src/components/Resizable.scss

```
@@ -1,5 +1,5 @@
 .ppd-resizable {
-  --inner-border-width: var(--border-width, 5px);
+  --inner-border-width: var(--border-width, 1px);
```

```suggestion
  --inner-border-width: var(--border-width, 5px);
```

Don't default value and I change the component caller props to change border width of `EditorZone` component.